docs/Outbound.md

```diff
@@ -40,10 +40,19 @@ Default: false. Switch to true to enable TLS for outbound mail when the
 remote end is capable.
 
 This uses the same `tls_key.pem` and `tls_cert.pem` files that the `tls`
-plugin uses. See the [tls plugin
+plugin uses, along with other values in `til.ini`. See the [tls plugin
```

Comment:
Fixed.

Only question about this I really have is, is `smtp_forward` and proxy
really "outbound". Do they need split off into a `[local]` section?

On Mon, Feb 22, 2016 at 8:57 PM, Matt Sergeant <helpme@gmail.com> wrote:

> Clearly too much Reddit r/til today ;)
>
> On Feb 22, 2016, at 8:53 PM, Matt Simerson <notifications@github.com>
> wrote:
>
> In docs/Outbound.md
> <https://github.com/haraka/Haraka/pull/1357#discussion_r53726158>:
>
> > @@ -40,10 +40,19 @@ Default: false. Switch to true to enable TLS for outbound mail when the
> >  remote end is capable.
> >
> >  This uses the same `tls_key.pem` and `tls_cert.pem` files that the `tls`
> > -plugin uses. See the [tls plugin
> > +plugin uses, along with other values in `til.ini`. See the [tls plugin
>
> typo? tls.ini
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/haraka/Haraka/pull/1357/files#r53726158>.
>
>